control/control.CAASP.xml

```diff
@@ -275,6 +275,97 @@ textdomain="control"
         <id>none_role</id>
 
         <order config:type="integer">300</order>
+
+      </system_role>
+
+      <system_role>
+        <id>micro_os_role</id>
+
+	<order config:type="integer">400</order>
+
+        <software>
+            <default_patterns>SUSE-MicroOS SUSE-MicroOS-hardware SUSE-MicroOS-apparmor</default_patterns>
+        </software>
+        <partitioning>
+            <expert_partitioner_warning config:type="boolean">true</expert_partitioner_warning>
+
+            <proposal>
+                <lvm config:type="boolean">false</lvm>
+                <proposal_settings_editable config:type="boolean">false</proposal_settings_editable>
+                <root_subvolume_read_only config:type="boolean">true</root_subvolume_read_only>
+            </proposal>
+
+            <volumes config:type="list">
+                <!-- The / filesystem -->
+                <volume>
+                    <mount_point>/</mount_point>
+                    <!-- Default == final, since the user can't change it -->
+                    <fs_type>btrfs</fs_type>
+                    <desired_size config:type="disksize">25 GiB</desired_size>
+                    <min_size config:type="disksize">16 GiB</min_size>
+                    <max_size config:type="disksize">30 GiB</max_size>
```

Comment:
is there reason for max size? There are no other mp, so unlimited should be fine not?